apps/nextjs/src/app/api/health/live/route.ts

```diff
@@ -0,0 +1,91 @@
+import { performance } from "perf_hooks";
+
+import { db } from "@homarr/db";
+import { logger } from "@homarr/log";
+import { handshakeAsync } from "@homarr/redis";
+
+export async function GET() {
+  const timeBeforeHealthCheck = performance.now();
+  const response = await executeAndAggregateAllHealthChecksAsync();
+  logger.info(`Completed healthcheck after ${performance.now() - timeBeforeHealthCheck}ms`);
+
+  if (response.status === "healthy") {
+    return new Response(JSON.stringify(response), {
+      status: 200,
+    });
+  }
+
+  return new Response(JSON.stringify(response), {
+    status: 500,
+  });
+}
+
+const executeAndAggregateAllHealthChecksAsync = async (): Promise<{
+  healthChecks: Record<string, object>;
+  status: "healthy" | "unhealthy";
+}> => {
+  const healthChecks = [
+    executeHealthCheckSafelyAsync("database", async () => {
+      // sqlite driver does not support raw query execution. this is for a heartbeat check only - it doesn't matter if data is returned or not
+      await db.query.serverSettings.findFirst();
+      return {};
+    }),
+    executeHealthCheckSafelyAsync("redis", async () => {
+      await handshakeAsync();
+      return {};
```

Comment:
Is there a reason it returns an empty object?

Reply:
Yes, because this method could return some data - but currently it does not. We could return additional data that is important for the healtcheck. I think we can leave it empty for now